draft-dt-tls-external-psk-guidance.md

```diff
@@ -274,6 +256,22 @@ pair-wise shared keys to achieve this. As another example, some systems require
 application-specific information in either PSKs or their identities. Identities may sometimes need to be routable,
 as is currently under discussion for EAP-TLS-PSK.
 
+
+# Recommendations for External PSK Usage
+
+
+Applications MUST use external PSKs that adhere to the following requirements:
+
+1. Each PSK MUST be derived from at least 128 of entropy and MUST be at least 128-bits long unless the TLS handshake is being used with a separate key establishment mechanism such as a Diffie-Hellman exchange. This recommendation protects against passive attacks using exhaustive search of the PSK.
+2. Each PSK MUST NOT be shared between with more than two logical nodes. As a result, an agent that acts as both a client and a server MUST use distinct PSKs when acting as the client from when it is acting as the server. This prevents redirection attacks.
+3. Nodes SHOULD use external PSK importers {{!I-D.ietf-tls-external-psk-importer}} when configuring PSKs for a pair of TLS client and server. {{!RFC8446}} mandates that each external PSK can only be used with a single hash function. {{!I-D.ietf-tls-external-psk-importer}} overcomes this limitation by producing a set of candidate PSKs, each of which are 
```

Comment:
The second and third sentences in this bullet seem like too much information for the requirement. Can we please revert to the previous text?